src/core/render/index.js

```diff
@@ -48,22 +46,35 @@ function renderMain(html) {
   }
 
   this._renderTo('.markdown-section', html);
+
   // Render sidebar with the TOC
   !this.config.loadSidebar && this._renderSidebar();
 
-  // Execute script
-  if (
-    this.config.executeScript !== false &&
-    typeof window.Vue !== 'undefined' &&
-    !executeScript()
-  ) {
-    setTimeout(_ => {
-      const vueVM = window.__EXECUTE_RESULT__;
-      vueVM && vueVM.$destroy && vueVM.$destroy();
-      window.__EXECUTE_RESULT__ = new window.Vue().$mount('#main');
-    }, 0);
-  } else {
-    this.config.executeScript && executeScript();
+  // Execute markdown <script>
+  if (this.config.executeScript || 'Vue' in window) {
+    executeScript();
+  }
+
+  // Handle Vue content not handled by markdown <script>
+  if ('Vue' in window) {
+    const mainElm = document.querySelector('#main') || {};
+    const childElms = mainElm.children || [];
+
+    for (let i = 0, len = childElms.length; i < len; i++) {
+      const elm = childElms[i];
+      const isValid = ['SCRIPT'].indexOf(elm.tagName) === -1;
```

Comment:
This could be better: 

```js
const isValid = elm.tagName !== 'SCRIPT';
```

Reply:
I was using `indexOf` in expectation that there might be other tags that should be skipped (e.g. `<br>`, `<link>`, `<meta>`). At the time, `<script>` was the only one Vue was complaining about. With this implementation it's obvious how to skip multiple tags if/when we need to do so.

Happy to make the change though.

Reply:
if we are going to have other tags as well. It does make sense to go with arrays. 

I was thinking of using `Sets` as they are a bit faster.   

thoughts ?

Reply:
@anikethsaha: No reason to focus on speed improvements for fixed arrays with one item. Also, we can't use `Set` natively due to our browser compatibility list without adding a significant amount of polyfill code.

Reply:
Make sense